hack/compare-events.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Script to compare against upstream version for differences
```

Comment:
I added this script to detect differences between upstream and this chart, it produced a diff, could we check what might be happening?

Reply:
sure